app/facades/locations_facade.rb

```diff
@@ -46,11 +46,11 @@ def meters(index)
     end
 
     def minutes(index)
-      (proximity_json[:rows][0][:elements][index][:duration][:text]).to_f
+      proximity_json[:rows][0][:elements][index][:duration][:text].to_f
     end
 
     def seconds(index)
-      (proximity_json[:rows][0][:elements][index][:duration][:value]).to_f
+      proximity_json[:rows][0][:elements][index][:duration][:value].to_f
     end
```

Comment:
Not super relevant to this PR - just cleanup of syntax that was missed.

Reply:
Great, thanks for clarifying what happened here.